storage/pipeline/piece/piece_info.go

```diff
@@ -24,7 +24,7 @@ type PieceDealInfo struct {
 	DealID       abi.DealID
 	DealProposal *market.DealProposal
 
-	// Common deal info
+	// Common deal info, required for all pieces
```

Comment:
Add a note or make an issue that eventually this shouldn't be required

Reply:
https://github.com/filecoin-project/lotus/issues/11237